src/tiktok_uploader/upload.py

```diff
@@ -762,8 +764,9 @@ def _post_video(driver) -> None:
         driver.execute_script('document.querySelector(".TUXButton--primary").click()')
 
     # waits for the video to upload
+    tmp1 = "/html/body/div[6]/div/div/div[3]/button[2]"
     post_confirmation = EC.presence_of_element_located(
-        (By.XPATH, config["selectors"]["upload"]["post_confirmation"])
+        (By.XPATH, tmp1) # config["selectors"]["upload"]["post_confirmation"] was instead of tmp1
```

Comment:
Same above comment, you should change value in config file instead of `tmp` variable

Reply:
@lavanphat  fixed